htsohm/pseudo_material.py

```diff
@@ -104,3 +108,36 @@ def volume(self):
 
     def number_density(self):
         return len(self.atom_sites) / self.volume()
+
```

Comment:
How are these different from the ones defined above? Do we really need two sets?